ports/linux/bip6.c

```diff
@@ -284,7 +284,7 @@ uint16_t bip6_receive(
     int max = 0;
     struct timeval select_timeout;
     struct sockaddr_in6 sin = { 0 };
-    BACNET_IP6_ADDRESS addr = { { 0 } };
+    BACNET_IP6_ADDRESS addr = { { 0 }, 0 };
```

Comment:
It is easier to maintain declaration initial value as `= { 0 };` pattern - is that possible in this case?

Reply:
> It is easier to maintain declaration initial value as `= { 0 };` pattern - is that possible in this case?

Yeah that is better way in this. I do not know why I miss that. Thanks for review. If anyone else is wondering what this means: we are currently doing double braces like { { 0 } } compiler things we might actually want field initialize. Newer compilers does not however give warning if we do just { 0 } which is current idiom in C to do struct initializing.

I add couple more place where I found this same thing. Can be checked from commit after I push.